x-pack/plugins/maps/server/plugin.ts

```diff
@@ -77,7 +77,11 @@ export class MapsPlugin implements Plugin {
 
     home.sampleData.addAppLinksToSampleDataset('ecommerce', [
       {
-        path: getFullPath('2c9c1f60-1909-11e9-919b-ffe5949a18d2'),
+        sampleObject: {
+          type: 'map',
```

Comment:
nit: Would you mind reusing `MAP_SAVED_OBJECT_TYPE` from https://github.com/elastic/kibana/blob/33fd1bdff0556e899e37e6885b3c11f7c4bc33ac/x-pack/plugins/maps/common/constants.ts#L28?

It's already imported in this file too.